docs/user/azure/dnszone.md

```diff
@@ -0,0 +1,31 @@
+# Public DNS Zone
+
+Azure's DNS Zone service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup
+for the cluster to the outside world. To use OpenShift, you must have created a public DNS zone in Azure in
+the same subscription as your OpenShift cluster. You must also ensure the zone is "authoritative" for the domain. There are
+two ways to do this outlined below: root domain and subdomain. A root domain is `openshiftcorp.com`. A subdomain is of
+the form `clusters.openshiftcorp.com`.
+
+The below sections identify how to ensure your hosted zone is authoritative for a domain.
+
+## Step 1: Acquire/Identify Domain
+
+You may skip this step if using an existing domain and registrar. You will move the authoritative DNS to Route53 or
+submit a delegation request for a subdomain in a later step.
```

Comment:
Route 53?  Copy/paste from AWD docs which should be replaced with whatever Azure calls their DNS service ;).

Reply:
https://github.com/openshift/installer/compare/fc6532c8637a2c37f4c0a028da1a5a430ac19302...3bb0cfefaf513837f94ff94c1bc59e047421323b